pkg/apis/autoscaling/register.go

```diff
@@ -83,6 +83,19 @@ const (
 	// scale-to-zero-pod-retention-period global setting.
 	ScaleToZeroPodRetentionPeriodKey = GroupName + "/scaleToZeroPodRetentionPeriod"
 
+	// MetricAggregationAlgorithm is the annotation that can be used for selection
+	// of the algorithm to use for averaging metric data in the Autoscaler.
+	// Since autoscalers are a pluggable concept, this field is only validated
+	// for Revisions that are owned by Knative Pod Autoscaler.
+	// Possible values for KPA are:
+	// - empty/missing or "linear" — linear average over the whole
+	//   metric window (default);
+	// - weightedExponential — weighted average with exponential decay.
+	//   KPA will compute the decay multiplier automatically and it is capped
+	//	 at 0.2 from below. This algorithm might not utilize all the values
```

Comment:
The word "below" confused me for a bit here. I was initially reading it as referring to an algorithm lower down in the file, but I think we mean 0.2 is the lower bound of the multiplier? Maybe we could use some other form of words if so, like "with a lower bound of 0.2" or just "at least 0.2"

Reply:
Done.